getdgraph.sh

```diff
@@ -173,7 +173,14 @@ printf "%b" "$RESET"
 		exit 1;
 	fi
 
-	dgraph=$(grep -m 1 "$install_path"/dgraph  /tmp/"$checksum_file" | grep -E -o '[a-zA-Z0-9]{64}')
+	if grep -Fxq "$install_path"/dgraph /tmp/"$checksum_file"
+	then
+		# for versions < 22, old team was hardcoding install paths in SHA file
```

Comment:
let's just say here that we have modified our release process and an example of the structure would be useful.

Reply:
+1 ^